system-overview/cryptography.md

```diff
@@ -0,0 +1,115 @@
++++
+title = "Cryptography"
+weight = 50
+template = "doc.html"
++++
+
+Cryptography is central to the operation of Urbit. Here we give an overview of
+how it is utilized.
+
+There are two categories of keys and five components of the system involved with
+cryptography on Urbit. We first summarize the two categories of keys and how
+they are utilized by each ship type, then cover how different parts of Urbit are
+involved in cryptography.
+
+### Types of keys
+
+The two categories of keys are your Azimuth/Ethereum keys and your networking
+keys. In both cases, these are public/private key pairs utilized for public key
+cryptography.
+
+#### Azimuth keys
+
+Your Urbit ID exists as an ERC-721 non-fungible token on the Ethereum
+blockchain, and as such is contained in a wallet whose private key you possess.
+If you are are utilizing a [master
+ticket](@/docs/azimuth/azimuth.md#master-ticket), this private key is derived
+from a seed, which is what you use to login to
+[Bridge](@/docs/glossary/bridge.md). Otherwise, you have generated the key by
+some other process, of which there are too many to list here. Besides the
+private key which unlocks your ownership wallet address, you may have a few
+other private keys which unlock a wallet that corresponds to your ship's
+[proxies](@/docs/glossary/proxies.md). We refer collectively to these keys as
+your _Azimuth keys_.
+
+Only [planets](@/docs/glossary/planet.md), [stars](@/docs/glossary/star.md), and
+[galaxies](@/docs/glossary/galaxy.md) have Azimuth keys.
+[Moons](@/docs/glossary/moon.md) and [comets](@/docs/glossary/comet.md) do not,
+as they do not utilize the Ethereum blockchain.
+
+It is important to note that no Azimuth keys are stored anywhere within your
+ship's [pier](@/docs/glossary/pier.md) - Ethereum and Urbit ID are entirely
+separate entities from Urbit itself, and so you lose access to your Azimuth
+private keys there is no way to retrieve them somehow from your ship.
+
+For more information on the usage of these keys and the associated proxies, see
+the [Azimuth documentation](@/docs/azimuth/azimuth.md).
+
+#### Networking keys
+
+All communications in Urbit over the [Ames](@/docs/glossary/ames.md) networking
+are end-to-end encrypted, and thus your ship stores its own public/private pair
+of _networking keys_ utilized for encryption and authentication. Networking keys
+for all ship types are stored within the ship's Jael vane.
```

Comment:
Are we fine to say "vane" here? At the very least this should probably link to the glossary entry or overview page for Jael.